pkg/apis/v1alpha1/vpc_types.go

```diff
@@ -39,8 +39,10 @@ type VPCStatus struct {
 	// NSX VPC Policy API resource path.
 	NSXResourcePath string `json:"nsxResourcePath"`
 	// Default SNAT IP for private Subnets
-	DefaultSNATIP string            `json:"defaultSNATIP"`
-	CIDRsUsage    VPCCIDRsUsageInfo `json:"cidrsUsage"`
+	DefaultSNATIP string `json:"defaultSNATIP"`
+	// NSX PolicyPath for AVI private Subnet
+	LBSubnetPath string            `json:"lbSubnetPath"`
```

Comment:
Could we add the LBSubnetCIDR too? I assume there are cases we expect customers to manually create SecurityPolicy with the LB CIDR to allow traffic from LB to workloads?

Reply:
OK, add LBSubnetCIDR.